tools/test.py

```diff
@@ -244,12 +240,6 @@ def main():
                 if target.is_PSA_secure_target:
```

Comment:
Will `mbed test` be run on secure targets? If not then can we remove this?

Reply:
https://github.com/ARMmbed/mbed-os/pull/11685#discussion_r343076062